treeherder/model/derived/jobs.py

```diff
@@ -431,6 +431,20 @@ def get_performance_series_summary(self, interval_seconds):
 
         return series_summary
 
+    def get_platforms(self, interval_seconds):
```

Comment:
get_performance_platforms would be a better name, since we're getting platforms for performance data here specifically.